rpm/SOURCES/etc/cron.d/cron-logrotate-contextAdapter-size

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of the iotagent-context-adapter.
```

Comment:
I saw some references to `iotagent-context-adapter`in this file... I guess you mean `context-adapter` right? :)